Dockerfile

```diff
@@ -35,7 +35,13 @@ RUN mkdir -p $TMP_DIR \
 COPY Gemfile \
      Gemfile.lock ./
 
-RUN bundle --without test development
+ENV WEB_CONCURRENCY=2
```

Comment:
Is this consistent with the default `WEB_CONCURRENCY` value, or is this a change in behavior? If we proceed with this change, then I think it needs to be clearly called out in the PR description - and it would be useful to also describe the motivation, so we can understand the context around why this change was made when trying to understand it later.